ci/run_graphing_test.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "Run functional test playbooks"
+  hosts:  "{{ cifmw_target_hook_host | default('localhost')  }}"
+  gather_facts: true
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: "Run Graphing Console UI tests"
+      ansible.builtin.import_role:
+        name: telemetry_graphing
+
```

Comment:
The typical approach is to use "ignore_errors: true" so that all the tests get run, then the report_result will check for failures.